R-package/README.md

```diff
@@ -5,15 +5,44 @@
 ### Contents
 
 * [Installation](#installation)
+    - [Installing the CRAN package](#installing-the-cran-package)
+    - [Installing from Source with CMake](#installing-from-source-with-cmake)
+    - [Installing a GPU-enabled Build](#installing-a-gpu-enabled-build)
+    - [Installing Precompiled Binaries](#installing-precompiled-binaries)
+    - [Installing from a Pre-compiled lib_lightgbm](#lib_lightgbm)
 * [Examples](#examples)
 * [Testing](#testing)
-* [Preparing a CRAN Package and Installing It](#preparing-a-cran-package-and-installing-it)
+* [Preparing a CRAN Package](#preparing-a-cran-package)
 * [External Repositories](#external-unofficial-repositories)
 * [Known Issues](#known-issues)
 
 Installation
 ------------
 
+For the easiest installation, go to ["Installing the CRAN package"](#installing-the-cran-package).
```

Comment:
I think we should say here that **the recomended way is CMake + VS** due to performance reasons.

Reply:
I disagree with that being "the recommended way". I don't think we should assume that the average user of the R package would prefer efficiency of the library over ease of installation. The CMake path is an order of magnitude more difficult than the CRAN one.

I did note the performance expectations in the next line

> If you experience any issues with that, try ["Installing from Source with CMake"](#install). This can produce a more efficient version of the library on Windows systems with Visual Studio.

Reply:
cc @guolinke and @Laurae2

Reply:
I won't merge this until this thread is answered. Gently ping @guolinke

Reply:
sorry for missing this. For cran and R, I think easy-to-install is preferred by users. So we can add a note about cmake+vs is more efficiency in windows.

Reply:
ok great, thanks! Then I think this is ok as of I've written it, with the note on the the choice of cmake + VS for a more efficient version